EMS/helpers/tests/Unit/Standard/LocaleTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EMS\Helpers\Tests\Unit\Standard;
+
+use EMS\Helpers\Standard\Locale;
+use PHPUnit\Framework\TestCase;
+
+class LocaleTest extends TestCase
+{
+    /**
+     * @dataProvider provideLocaleInput
+     */
+    public function testShortLocale(?string $input, ?string $default, string $expected): void
+    {
+        $this->assertEquals($expected, Locale::short($input, $default));
+    }
+
```

Comment:
What about the test 'toto''tata' => exception?

Reply:
Fixed